std/os.zig

```diff
@@ -2203,6 +2235,16 @@ pub fn argsAlloc(allocator: *mem.Allocator) ![]const []u8 {
 }
 
 pub fn argsFree(allocator: *mem.Allocator, args_alloc: []const []u8) void {
+    if (builtin.os == Os.wasi) {
+        const last_item = args_alloc[args_alloc.len - 1];
+        const last_byte_addr = @ptrToInt(last_item.ptr) + last_item.len + 1; // null terminated
```

Comment:
If this has a +1 doesn't the other function need a -1?

Reply:
The strings in `argv_buf`, that is written to by the `args_get` function, are contiguous null terminated c-strings. My assumption here is that `mem.toSlice` doesn't include the final null-terminated buffer and I've added it myself here for the final string.

Reply:
Ah, I see.